lib/pbench/test/unit/server/test_endpoint_configure.py

```diff
@@ -58,9 +58,9 @@ def check_config(self, client, server_config, host, my_headers={}):
                 "register": f"{uri}/register",
                 "login": f"{uri}/login",
                 "logout": f"{uri}/logout",
-                "user": f"{uri}/user/",
+                "user_x": f"{uri}/user/x",
```

Comment:
Yeah, really don't like "user_x", and "x" is too generic for a client to just assume it should substitute "x" with the actual username. Again, the dashboard won't use the upload API and doesn't currently GET/DELETE/PUT users but even as a temporary lint this bugs me.

Reply:
Moved to #2287.